### PR TITLE
Event Hub : Split swagger in to entities

### PR DESCRIPTION
**Pull request by v-Ajnava**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [X] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by raych1** [comment]
Hi @v-Ajnava, please fix the CI check failures. And can you share what's the motivation to split the swagger?

**Comment by KrisBash** [comment]
@v-Ajnava Can you help clarify what is changed in this PR beyond the split into multiple files? That makes it impossible to diff changes for review.

**Comment by v-Ajnava** [comment]
Hey @KrisBash , there are no changes other than splitting the swagger. 

@raych1 , the motivation to split swagger to make it easy for working on the changes and to get client with features from both 2018-01-01-preview and 2017-04-01 swagger (except namespaces)

File: specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/AuthorizationRules.json [hunk 1]
```diff
@@ -0,0 +1,886 @@
...
+            "$ref": "../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/NamespaceNameParameter"
+          },
+          {
+            "$ref": "../../common/v1/definitions.json#/parameters/EventHubNameParameter"
+          },
+          {
+            "$ref": "#/parameters/AuthorizationRuleNameParameter"
```
Comment (raych1) [inline specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/AuthorizationRules.json:733]: This reference should be updated to from common file?

**Comment by raych1** [comment]
@v-Ajnava , there's a parameter reference not updated to new location from common file which failed the model validation check.

**Review by KrisBash — APPROVED** [review]
Signing off for ARM, no API changes are expected in this update

**Review by raych1 — APPROVED** [review]

**Pull request merged**